site/content/docs/features/config-file.md

```diff
@@ -1 +1,632 @@
+
+_**ReverseProxy**_  
+The reverse proxy to use in front of the .NET Core web server, Kestrel. This parameter can be set to "nginx" (the default), or it can be set to "none" to use Kestrel as the front facing web server. 
+
+_**EnhancedHealthReporting**_  
+Enables enhanced health reporting, which provides free, real-time application and operating-system monitoring of the instances and other resources in your environment. Valid values are "Enhanced" (the default) and "Basic".
+
+This parameter is valid only if `ManagedActionsEnabled` under `ElasticBeanstalkManagedPlatformUpdates` (shown above) is set to `false`.
+
+_**HealthCheckURL**_  
+Customize the load balancer health check to ensure that your application is in a good state in addition to the web server. The default value for this parameter is "/".
+
+This parameter is valid only if `EnvironmentType` (shown above) is set to "LoadBalanced".
+
+_**ElasticBeanstalkRollingUpdates**_  
+A collection of parameters that define how AWS Elastic Beanstalk performs rolling updates for the environment in which your application operates. For more information, see [Rolling updates](https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/using-features.rollingupdates.html) in the [AWS Elastic Beanstalk Developer Guide](https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/).
+
+This object is valid only if `EnvironmentType` (shown above) is set to "LoadBalanced".
+
+By default, rolling updates are disabled. To enable rolling updates, set `RollingUpdatesEnabled` to `true`. Then use the other parameters of this object (`RollingUpdateType`, etc.) to configure the behavior. These parameters are described as follows:
+
+* `RollingUpdateType`  
+The type of rolling update to implement.  
+-- Use a value of "Time" (the default) for time-based rolling updates. These updates wait a certain amount of time between batches. The amount of time is specified by the `PauseTime` parameter shown below.  
+-- Use a value of "Health" for health-based rolling updates. These updates wait for new instances to pass health checks before moving on to the next batch.  
+-- Use a value of "Immutable" for immutable updates. These updates launch a full set of instances in a new Auto Scaling group.
+
+* `MaxBatchSize`  
+The number of instances included in each batch of the rolling update. The valid range of values is 1 through 10000.
+
+* `MinInstancesInService`  
+The minimum number of instances that must be in service within the Auto Scaling group while other instances are terminated. The valid range of values is 0 through 9999.
+
+* `PauseTime`  
+The amount of time that the Elastic Beanstalk service waits after completing updates to one batch of instances before it continues with the next batch. The format of this parameter must be "PT#H#M#S" where each '#' is the number of hours, minutes, and seconds, each of which is optional.

```

Comment:
What is the default?

Reply:
The defaults can be found here https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/command-options-general.html#command-options-general-autoscalingupdatepolicyrollingupdate

Reply:
Added link.